shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/utils/MobileLogger.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.kotlinconf.utils
+
+import io.ktor.client.plugins.logging.Logger
+
+expect object MobileLogger : Logger
```

Comment:
IIRC, Kotlin recommends to use expect/actual functions instead of classes.
So,
```kotlin
internal val mobileLogger = getMobileLogger()
expect fun getMobileLogger(): Logger
``` 
is better

Reply:
thanks, fixed